test/non-regression.js

```diff
@@ -1135,4 +1135,38 @@ describe("verify", function () {
       []
     )
   });
+
+  it("line comment space-in-parens #124", function () {
+    verifyAndAssertMessages(
+      [
+        "React.createClass({",
+          "render() {",
+             "// return (",
+             "//   <div />",
+             "// ); // <-- this is the line that is reported",
```

Comment:
Looks like this particular test still fails after the comment changes in babel because of `//_` where _ is the space. Not sure why yet - tokens and comments look fine.